CMakeLists.txt

```diff
@@ -19,8 +19,15 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 if(UNIX AND NOT APPLE AND NOT DEFINED OpenGL_GL_PREFERENCE)
   set(OpenGL_GL_PREFERENCE LEGACY)
 endif()
+
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-reorder")
```

Comment:
Please add a comment explaining why this is needed.

Reply:
The "-Wno-reorder" flag was inserted in the latest qmake project update (olive.pro):
https://github.com/olive-editor/olive/blob/master/olive.pro#L51

Reply:
File changed in commit 7d0a666.